pkg/controllers/disruption/emptynodeconsolidation.go

```diff
@@ -55,15 +56,17 @@ func (c *EmptyNodeConsolidation) ComputeCommand(ctx context.Context, disruptionB
 		if len(candidate.reschedulablePods) > 0 {
 			continue
 		}
-		if disruptionBudgetMapping[candidate.nodePool.Name] == 0 {
+		if disruptionBudgetMapping[candidate.nodePool.Name][v1beta1.DisruptionReasonUnderutilized] == 0 {
```

Comment:
@jonathan-innis and I were discussing this, and were hoping that this would be the Empty reason. If we treat this a the budget for underutilized, someone with a `consolidationPolicy: whenUnderutilized`, will never have a reason to set the `Empty` budget, even though this operation is purely for empty nodes.

Reply:
Yep I agree, lets set it to that.

Reply:
Alright moved it over. Added a test to validate the distinction between the two as well.